File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/DelingstallUtvalg.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class DelingstallUtvalg implements Serializable {


    private double dt;
    private double PREG_dt67soker;
    private double PREG_dt67virk;
    private List<Delingstall> delingstallListe = new ArrayList<Delingstall>();
}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/DelingstallUtvalg.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable

class DelingstallUtvalg : Serializable {
    var dt = 0.0
    var pREG_dt67soker = 0.0
    var pREG_dt67virk = 0.0
    var delingstallListe: List<Delingstall> = mutableListOf()
}